src/deadline/cinema4d_adaptor/Cinema4DClient/plugin/DeadlineCloudClient.pyp

```diff
@@ -12,7 +12,7 @@ for n in sys.path:
 
 # cinema4d doesn't use PYTHONPATH so explicitly load modules
 if 'openjd' not in sys.modules.keys():
-    python_path = os.getenv('DEADLINE_CLOUD_PYTHONPATH')
+    python_path = os.getenv('DEADLINE_CLOUD_PYTHONPATH', '')
```

Comment:
Hmm, should we have a `if not python_path` condition below to notify customers that  they need to set this variable?